tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "edv-client-tests",
+  "version": "0.0.1",
+  "description": "Tests for the edv-client.",
+  "main": ".eslintrc.js",
+  "dependencies": {},
+  "devDependencies": {
+    "crypto-ld": "^3.7.0",
```

Comment:
curious, why create a separate package.json for tests instead of installing these dependencies as dev dependencies of the root level package?

Reply:
We do have projects with separate test projects, but in this case it looks like devDepdencies contains the other test dependencies so I decided to go with that and to remove the `postinstall` for the script that installs the test dependencies.